src/EmailFactory.php

```diff
@@ -29,8 +29,8 @@ public function create(string $mailbox, Horde_Imap_Client_Data_Fetch $hordeEmail
         // Parse the message body
         $parser = new Parser();
         $parser->setText($hordeEmail->getFullMsg());
-        $htmlContent = utf8_encode((string) $parser->getMessageBody('html'));
```

Comment:
😢  I'm terribly ashamed to admit I didn't write tests in this package (testing something calling IMAP would mean mocking a lot of stuff which defeats the purpose of the tests, or putting in a lot of work into a setup with Vagrant + IMAP server). It was the kind of thing I left for later (this package is more of a prototype than a complete solution)

I'm tempted to try it the old way: merge it and see what happens :D